cloudsplaining/command/scan.py

```diff
@@ -37,6 +37,8 @@
 @click.option("-m", "--minimize", required=False, default=False, is_flag=True, help="Reduce the size of the HTML Report by pulling the Cloudsplaining Javascript code over the internet.")
 @click.option("-aR", "--flag-all-risky-actions", is_flag=True, help="Flag all risky actions, regardless of whether resource ARN constraints or conditions are used.")
 @click.option("-v", "--verbose", "verbosity", help="Log verbosity level.", count=True)
+@click.option("-f", "--filer-severity", "severity", help="Filter the severity of findings to be reported.", multiple=True,type=click.Choice(['HIGH', 'MEDIUM','LOW'], case_sensitive=False))
```

Comment:
Please add 'CRITICAL' and 'NONE' options to the choices list, to bring the possibly severity settings in line with CVSS v3 ([CVSS v3.1 Specification Document (first.org)](https://www.first.org/cvss/specification-document#Qualitative-Severity-Rating-Scale))

Typo: long option name should be `--filter-severity`